source/III.VC.CLEO/Fxt.cpp

```diff
@@ -24,6 +24,40 @@ CustomTextEntry::~CustomTextEntry()
 		delete m_pText;
 }
 
+static uint8_t get_utf8_bytes(uint8_t utf8)
+{
+	for (uint8_t i = 0; i < 6; i++) {
```

Comment:
This project is using "opening bracket in new line" style. Better keep it consistent.